requirements.txt

```diff
@@ -1,2 +1,3 @@
 bentoml
-docker
\ No newline at end of file
+docker
+cerberus
```

Comment:
should we lock the version for this @yubozhao?

Reply:
The operator doesn't really need Cerberus library. It just needs to define a validation schema json that's compatible with it. 

For docker, we can lock to the latest stable version as this deployment doesn't require anything too exotic.

Reply:
cc @dtemir

Reply:
Oh so we don't need Cerberus in requirements.txt?

Reply:
no we don't need to use cerberus at all

Reply:
oh so is the schema not needed? should I close the PR?